bin/cli.js

```diff
@@ -52,7 +70,10 @@ function initKnex(env) {
 
   var environment = commander.env || process.env.NODE_ENV;
   var defaultEnv = 'development';
-  var config = require(env.configPath);
+  var config =
```

Comment:
Would you consider replacing all vars in this file with let/const?

Reply:
I could, however there are many "var's" in the same scope and upper. I could do this in another PR to isolate the changes.

Reply:
Fair enough :)